drivers/w1/Kconfig.ds248x

```diff
@@ -0,0 +1,6 @@
+# Copyright (c) 2022, Caspar Friedrich <c.s.w.friedrich@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+config W1_DS248X
+	bool "DS2482/4 Single-Channel 1-Wire Master"
+	depends on I2C
```

Comment:
Add `default $(dt_compat_enabled,$(DT_COMPAT_MAXIM_DS248X))`

(See `drivers/counter/Kconfig.esp32` for example)

Reply:
Good idea, done.